services/gateway/apis/http-version3/methods/market.js

```diff
@@ -43,8 +43,13 @@ module.exports = {
 					$ref: '#/definitions/MarketPricesWithEnvelope',
 				},
 			},
+			503: {
+				description: 'Service Unavailable',
+				schema: {
+					$ref: '#/definitions/serviceUnavailable',
+				},
+			},
 		};
-		Object.assign(marketPricesSchema[this.swaggerApiPath].get.responses, response);
```

Comment:
should't this api return bad request if called with invalid params?

Reply:
Applied the suggested change to take care of unwanted params